apps/src/gamelab/constants.js

```diff
@@ -40,3 +40,27 @@ module.exports.AnimationCategories = {
   category_obstacles: 'Obstacles',
   category_all: 'All'
 };
+
+module.exports.SpritelabReservedWords =
```

Comment:
Is it possible to format this with one reserved word per line? I believe that would be easier to parse in case we want to check if a specific reserved word has been already used.
Also, does capitalization matter for these words?

Reply:
Yes, I believe they are case-sensitive